api/errors/errors.go

```diff
@@ -83,6 +80,12 @@ var ErrInvalidReceiverAddress = errors.New("invalid hex receiver address provide
 // ErrTransactionNotFound signals that a transaction was not found
 var ErrTransactionNotFound = errors.New("transaction not found")
 
+// ErrTransactionsNotFoundInPool signals that no transaction was not found in pool
+var ErrTransactionsNotFoundInPool = errors.New("transactions not found in pool")
```

Comment:
I cannot remove it completely. As discussed, it's better to return an error in the case of `last nonce` because you cannot differentiate between `0` and `null` otherwise.